plugins/riemann-controller/riemann_controller/resources/manager.config

```diff
@@ -77,7 +77,9 @@
 
 (defn- amqp-connection-conf
   []
-  {:host (or (System/getenv "RABBITMQ_HOST") "localhost")})
+  {:host (or (System/getenv "RABBITMQ_HOST") "localhost")
+   :username (or (System/getenv "RABBITMQ_USER") "testuser")
```

Comment:
We should probably check that the user and password are set in the riemann's create.sh file before copying its systemd env vars file. Alternatively, @dankilman, do you think there is a nice way for us to output an error in riemann if these values are not set?